project-base/storefront/cypress/tsconfig.json

```diff
@@ -6,9 +6,10 @@
         "baseUrl": ".",
         "skipLibCheck": true,
         "strict": true,
+        "jsx": "preserve",
         "forceConsistentCasingInFileNames": true,
         "incremental": true,
         "esModuleInterop": true
     },
-    "include": ["**/*.ts", "**/*.cy.ts"]
+    "include": ["**/*.ts", "**/*.cy.ts", "../graphql/generated/index.tsx"]
```

Comment:
Im not sure if that makes sense? Are we expecting more tsx files here?

Reply:
In case there are some new files are you sure everyone will know about adding this file also to this config? I don't see reason why to add files individually.

Reply:
That is a fair point. I wanted to limit the leak of files from storefront to cypress, but if we do not care about that, then I have no problem with it.